src/components/Pricing/index.jsx

```diff
@@ -0,0 +1,145 @@
+'use client'
+
+import { useState } from 'react'
+import PricingCard from './PricingCard'
+import { initiateSubscription } from '@/utils/stripe'
+import { useRouter } from 'next/navigation'
+import { useFirebaseAuthContext } from '@/contexts/firebaseAuthContext'
+import { useFirebaseAuth } from '@/firebase/auth'
+import { useToast } from '@/components/UI/use-toast'
+
+const EMAIL_SUPPORT = 'Email support'
+
+const comingSoonFeatures = [
+  ['Access to US & UK templates', EMAIL_SUPPORT],
+  [
+    'Access to US, UK+ more premium contract templates',
+    'AI-based contract drafting',
+    'Multi-user access (Up to 3 users)',
+    'Integration with e-signature platform',
+    'Export contracts',
+    EMAIL_SUPPORT,
+  ],
+  [
+    'Access to US, UK+ more premium contract templates',
+    'AI-based contract drafting',
+    'Multi-user access (Up to 5 users)',
+    'Integration with e-signature platform',
+    'Export contracts',
+    EMAIL_SUPPORT,
+  ],
+]
+
+export default function Products({ plans }) {
+  const { authUser } = useFirebaseAuthContext()
+  const firebaseAuth = useFirebaseAuth()
+  const { push } = useRouter()
+
+  const [activeTab, setActiveTab] = useState('month')
+  const [selectedPlan, setSelectedPlan] = useState(plans[0]?.id)
```

Comment:
can you make selectedPlan as current plan of user?

Reply:
Makes sense. I'll change `selectedPlan` to `currentPlan` as per your suggestion.

Reply:
Changed.